examples/reachable.myl

```diff
@@ -9,6 +9,6 @@ DO
                               EMIT Edge.dst AS addr]);
     Delta = DIFF(NewlyReachable, Reachable);
     Reachable = UNIONALL(Delta, Reachable);
-WHILE [*COUNTALL(Delta) > 0];
+WHILE [FROM COUNTALL(Delta) AS size EMIT *size > 0];
```

Comment:
I almost suggested we disallow the `*` if `x` is already in the `from_args`, but that probably breaks the case of multiple invocations of `*x.avg, *x.std` .. right?